lib/tool/code_interpreter.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Tool
+  class CodeInterpreter < Base
```

Comment:
Do you think it makes sense to call this `RubyCodeInterpreter`? Of course we won't be able to execute many languages, but I could see us writing a Shell, JS, or other kinds of code interpreters.

Reply:
@rickychilcott yes, I think that's a good point.